src/Geometry.h

```diff
@@ -34,9 +34,11 @@ struct fsr_data {
   /** The FSR ID */
   int _fsr_id;
 
-  /** Characteristic point in Universe 0 that lies in FSR */
+  /** Characteristic point in Root Universe that lies in FSR */
   Point* _point;
```

Comment:
I just realized that the ``fsr_data`` struct does not have a destructor. This means that the ``_point`` and ``_centroid`` attributes will just be left dangling after an ``fsr_data`` is deleted from the ``Geometry`` (*e.g.*, when running many simulations during a parametric study).